frontend/tests/unit/specs/aquifers/components/View.spec.js

```diff
@@ -107,7 +107,7 @@ const aquiferFixture = {
 }
 
 describe('View Component', () => {
-  let fetch = null
+  let fetch
```

Comment:
Why was fetch changed to uninitialized instead of null?

Reply:
reverted